sdk/metrics-testing/src/test/java/io/opentelemetry/sdk/testing/assertj/metrics/MetricAssertionsTest.java

```diff
@@ -90,8 +93,14 @@
               // Points
               Collections.emptyList()));
 
+  private static final DoubleExemplar DOUBLE_EXEMPLAR =
+      DoubleExemplar.create(Attributes.empty(), 0, "span", "trace", 1.0);
+
   private static final DoublePointData DOUBLE_POINT_DATA =
-      DoublePointData.create(1, 2, Attributes.empty(), 3.0);
+      DoublePointData.create(1, 2, Attributes.empty(), 3.0, Collections.emptyList());
+
+  private static final DoublePointData DOUBLE_POINT_DATA_WITH_EXEMPLAR =
+      DoublePointData.create(1, 2, Attributes.empty(), 3.0, Arrays.asList(DOUBLE_EXEMPLAR));
```

Comment:
`Collections.singletonList` - not a big deal but may as well make intellij not mark this out

Reply:
Ah, I'm using vscode.